src/seedu/addressbook/data/person/Address.java

```diff
@@ -26,6 +33,22 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = address;
+        
+        //Split user-entered address
+        address.replace("/a","");
```

Comment:
As your friend pointed out, the parameter signs already removed once the value receive by this class.